kuma/static/js/wiki.js

```diff
@@ -883,4 +883,17 @@
         }
     }
 
+    // listens for post message from interactive editor
+    window.addEventListener('message', function(event) {
+        // get the interactive editor iframe
+        var iframe = document.querySelector('iframe.interactive');
```

Comment:
Let's not add an ID, that will only make it harder to adapt the code to allow multiple instances of the interactive examples per page.